From e3641d13d65a0f74818af9149dfb20a3bf228494 Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Fri, 11 Dec 2015 11:07:03 -0600
Subject: [PATCH 0770/5242] MLK-11999: ARM: dts: change the platform
 compatibility for i.MX6qp gpmi dts

commit  5d196637690dc849721a3a441fee9010c34f9b40 from
https://source.codeaurora.org/external/imx/linux-imx.git

change the compatibility for i.MX6qp gpmi dts file to distinguish it
from i.MX6q

Signed-off-by: Han Xu <b45815@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts b/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts
index 9830f79..ce910b6 100644
--- a/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts
+++ b/arch/arm/boot/dts/imx6qp-sabreauto-gpmi-weim.dts
@@ -22,6 +22,7 @@
 #endif
 
 &gpmi {
+	compatible = "fsl,imx6qp-gpmi-nand";
 	status = "okay";
 };
 
-- 
1.7.9.5

